ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -172,6 +172,45 @@ message DamlTransactionEntry {
 
   // The time used to derive contract ids
   google.protobuf.Timestamp submission_time = 6;
+
+  // The pre-computed transaction blinding information. Optional.
+  DamlTransactionBlindingInfo blinding_info = 7;
+}
+
+// A transaction's blinding information, consisting of disclosure and
+// divulgence info.
+//
+// "Disclosure" tells us which transaction nodes to communicate to which
+// parties.
+// See https://docs.daml.com/concepts/ledger-model/ledger-privacy.html
+//
+// "Divulgence" tells us which contracts to communicate to which parties
+// so that their participant nodes can perform post-commit validation.
+// Note that divulgence can also divulge e.g. contract IDs that were
+// created _outside_ this transaction.
+// See https://docs.daml.com/concepts/ledger-model/ledger-privacy.html#divulgence-when-non-stakeholders-see-contracts
+message DamlTransactionBlindingInfo {
+
+  // The disclosure of a transaction node to a set of local parties
+  message DisclosureEntry {
+    string node_id = 1;
+    repeated string disclosed_to = 2;
```

Comment:
Please use pluralized names for repeated fields (https://developers.google.com/protocol-buffers/docs/style#repeated_fields). E.g.:
```suggestion
    repeated string disclosed_to_local_parties = 2;
```

Reply:
Done.